common/changes/@uifabric/experiments/chicletUrlFix_2019-07-22-17-17.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Url is now a required prop and two test file are combined into one",
```

nit: can we put "Chiclet:" at the beginning of this change file? So

```suggestion
      "comment": "Chiclet: Url is now a required prop and two test files are combined into one",
```

---

And to generate change files from now on, try running 'npm run change' instead of 'rush change' - that'll give you the new format